docs/md/process.md

```diff
@@ -22,22 +22,22 @@ CRTP idiom. Moreover, derived classes specify what the intended type for elapsed
 times is.
 
 A process should expose publicly the following member functions whether needed
-(note that it isn't required to define a function unless the derived class wants
+(note that it is not required to define a function unless the derived class wants
 to _override_ the default behavior):
 
 * `void update(Delta, void *);`
 
   This is invoked once per tick until a process is explicitly aborted or ends
-  either with or without errors. Even though it's not mandatory to declare this
-  member function, as a rule of thumb each process should at least define it to
+  either with or without errors. Even though it is not mandatory to declare this
```

Comment:
If you want to break, I think the following is better then:

> Technically speaking, this member function is not strictly required. However, each process should at least define it to
  work _properly_.